src/org/parosproxy/paros/extension/option/OptionsParamView.java

```diff
@@ -453,24 +474,58 @@ public void setLargeResponseSize(int largeResponseSize) {
 		getConfig().setProperty(LARGE_RESPONSE_SIZE, largeResponseSize);
 	}
 
+	/**
+	 * @deprecated (TODO add version) Replaced by {@link #getFontSize(org.zaproxy.zap.utils.FontUtils.FontType)}.
+	 */
+	@Deprecated
 	public int getFontSize() {
 		return fontSize;

```

Comment:
We can delete the `fontSize`/`fontName` and call the new methods instead (e.g. `getFontSize(FontUtils.FontType.general)`).